wolfcrypt/src/port/devcrypto/wc_devcrypto.c

```diff
@@ -122,7 +122,7 @@ int wc_DevCryptoCreate(WC_CRYPTODEV* ctx, int type, byte* key, word32 keySz)
         case CRYPTO_SHA2_512_HMAC:
             ctx->sess.cipher = 0;
             ctx->sess.mac    = type;
-            ctx->sess.mackey    = (u_int8_t*)key;
```

Comment:
I seem to remember a build that was having issue with the uint8_t type. Cross compiling for use on an NXP i.MX8 device. If you jump back to commit dc8b796d1dd784e6c832420e3154a6b382307836 I had uint8_t type before, so just changing it back to uint8_t will break the build I had changed it for.

Reply:
Please just use 'byte' here, this lets the wolfssl headers take care of setting the unsigned char type.

Tested this with musl and with the compiler was using for NXP earlier:

First, duplicating the issue with "as is" code

```
$ CC="musl-gcc -I/home/jak/Documents/cryptodev-linux/" ./configure --enable-static --disable-shared --enable-devcrypto && make

wolfcrypt/src/port/devcrypto/wc_devcrypto.c:125:36: error: ‘u_int8_t’ undeclared (first use in this function); did you mean ‘uint8_t’?
             ctx->sess.mackey    = (u_int8_t *)key;
                                    ^~~~~~~~
                                    uint8_t
wolfcrypt/src/port/devcrypto/wc_devcrypto.c:125:36: note: each undeclared identifier is reported only once for each function it appears in
wolfcrypt/src/port/devcrypto/wc_devcrypto.c:125:46: error: expected expression before ‘)’ token
             ctx->sess.mackey    = (u_int8_t *)key;
```

Next trying with musl using 'byte'
```
$ CC="musl-gcc -I/home/jak/Documents/cryptodev-linux/" ./configure --enable-static --disable-shared --enable-devcrypto && make
...
...
...
  CCLD     examples/server/server
  CCLD     testsuite/testsuite.test
  CCLD     tests/unit.test
make[2]: Leaving directory '/home/jak/Documents/wolfssl'

```


Then trying the cross compile with the NXP toolchains (having the type be byte)

```
$ ./configure --host=aarch64-poky-linux --with-libz=$ZLIB_DIR --with-seco=$HSM_DIR --enable-caam=seco --enable-cmac --enable-aesgcm --enable-aesccm --enable-keygen CPPFLAGS="-DHAVE_AES_ECB -I$CRYPTODEV_DIR" --enable-devcrypto=seco --enable-curve25519 --enable-sha224 --enable-static --disable-shared --disable-filesystem && make
...
...
...
aarch64-poky-linux-ar: `u' modifier ignored since `D' is the default (see `U')
  CCLD     wolfcrypt/benchmark/benchmark
  CCLD     wolfcrypt/test/testwolfcrypt
make[2]: Leaving directory '/home/jak/Documents/wolfssl'
```